Support Volume Expansion (#32)
ofek/csi-gcs: docs/csi_compatibility.md

# CSI Specification Compatibility

This page describes compatibility to the [CSI specification](https://github.com/container-storage-interface/spec/blob/master/spec.md).

## Capacity

> :warning:
> **Google Cloud Storage does not allow to enforce capacity limits. Therefore, this driver is unable to provide capacity limit enforcement.**

Reviewer: ```suggestion
!!! warning "Important"
    Google Cloud Storage has no concept of capacity limits. Therefore, this driver is unable to provide capacity limit enforcement.
```
